src/main/java/org/spongepowered/api/Server.java

```diff
@@ -274,7 +274,7 @@
 
     /**
      * Shuts down the server, and kicks all players with the default kick
-     * message (the translation key {@code disconnect.closed}).
+     * message (the literal text "Server closed").
```

Comment:
Why not just remove this part and just say "with the default kick message"? I see no reason why a plugin or even Sponge itself shouldn't allow modifying the default kick message and it (should) actually get translated depending on the client.
Not completely sure why this is in this PR (seems unrelated), but I guess it's not a big problem